python/lsst/ts/xml/data/sal_interfaces/MTPtg/MTPtg_Events.xml

```diff
@@ -812,4 +812,37 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>MTPtg</Subsystem>
+    <EFDB_Topic>MTPtg_logevent_observatoryLocation</EFDB_Topic>
+    <Description>Location of the observatory.</Description>
+    <item>
+      <EFDB_Name>longitude</EFDB_Name>
+      <Description>Site longitude.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>degree</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>latitude</EFDB_Name>
+      <Description>Site latitude.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>degree</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>height</EFDB_Name>
+      <Description>Site height above see level.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>meter</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timeZone</EFDB_Name>
+      <Description>Time zone offset.</Description>
```

Comment:
What about summer time? timeZone changes every 6 months in Chile.

Reply:
If we have to update this when clocks change then we should consider just deleting this field (from both pointing components). It's not very interesting (especially compared to the other 3 fields, which also have the advantage of being static by any reasonable definition). I suppose the description could be clearer.